eventuate-core/src/it/scala/com/rbmhtechnology/eventuate/ReplicationIntegrationSpec.scala

```diff
@@ -162,8 +165,8 @@ trait ReplicationIntegrationSpec extends WordSpec with Matchers with MultiLocati
       val locationA = location("A")
       val locationB = location("B")
 
-      val endpointA = locationA.endpoint(Set("L1"), Set(replicationConnection(locationB.port, filters = Map("L1" -> new PayloadInequalityFilter("b1")))), filters = Map("L1" -> new PayloadInequalityFilter("a3")))
-      val endpointB = locationB.endpoint(Set("L1"), Set(replicationConnection(locationA.port, filters = Map("L1" -> new PayloadInequalityFilter("a1")))), filters = Map("L1" -> new PayloadInequalityFilter("b3")))
+      val endpointA = locationA.endpoint(Set("L1"), Set(replicationConnection(locationB.port, filters = Map("L1" -> new PayloadInequalityFilter("b1")))), sourceFilters(Map("L1" -> new PayloadInequalityFilter("a2"))))
+      val endpointB = locationB.endpoint(Set("L1"), Set(replicationConnection(locationA.port, filters = Map("L1" -> new PayloadInequalityFilter("a1")))), sourceFilters(Map("L1" -> new PayloadInequalityFilter("b2"))))
```

Comment:
Is there a special reason that `a3` and `b3` have been replaced by `a2` and `b2`?